The Global Content Widget (#2417)
umbraco/UmbracoDocs: Add-ons/Starter-Kit-TEMP/Widgets/index.md

Below you can see an example that uses the back and white feature, and has two logos.
![movePage.jpg](images/Logos.png)

### The Global Content Widget
![movePage.jpg](images/The-Global-Content-Widget.png)

> What The Global Content Widget does for you, is that it allows you to pick those templates you created, and use them on your pages.

Reviewer: If you read about the Global Content Widget before you know what Global Content is, I think we need to rephrase this first sentence a bit:

```suggestion
What this widget does for you, is that it allows you to pick pre-made widgets, also called Global Content.

To learn more about how to create Global Content, you can read the [Global Content](../Content-Types/Global-Content/index.md) article.
```

> - This can be done by clicking the add content button on your page
> - From here select Global Content
> - Then once that has been added, you have to navigate to where it says add and click this

Reviewer: ```suggestion
Once you've added the Global Content Widget to your page, follow these steps to select which Global Content you want to use:

- Click "Add"
```

> - Then a list pop's up on the right-hand side of the screen
> - From here you select global widgets (click the arrow next to it)
> - Then you will get to select, which of your pre-made widget templates you want to use

Reviewer: ```suggestion
- From the dialog in the right-hand side, expand the folder containing your Global Content (click the arrow next to it)
- Finally, select which of your pre-made widgets you want to add to your page
```

> keep in mind that in order for there, to be anything to select, you will have to have set-up some global content, in the global content node, which can be found in the section tree.

Reviewer: ```suggestion
Keep in mind that in order for there to be anything to select, you will have to have set-up some widgets using the [Global Content](../Content-Types/Global-Content/index.md) feature.
```

> Here you can learn how to [Create Global Content](../Content-Types/Global-Content/index.md).

Reviewer: I've added the link further up in the article.
```suggestion
```

This is a very useful tool if you have a widget that you use a lot and want to look the same on multiple pages.

> See below an example, of what it might look like. In this example, there is a hero widget which should appear on multiple pages therefore by selecting it with The Global Content Widget this can be the case:

Reviewer: ```suggestion
See below an example, of what it might look like. In this example, there is a Hero Widget which have been created as Global Content and can therefore by selected with The Global Content Widget.
```
